pkg/cli/token.go

```diff
@@ -104,6 +116,10 @@ func runTokenCmd(cmd *cobra.Command, opts *tokenCmdOptions, args []string) error
 	ctx := context.Background()
 	out := cmd.OutOrStdout()
 
+	if opts.explanation == "" {
```

Comment:
Should we validate the subject is not empty also? Just thinking of the case where someone didnt set the env var and didnt use the flag. Would be frustrating later when the token doesnt work.